shenyu-client/shenyu-client-motan/src/main/java/org/apache/shenyu/client/motan/MotanServiceEventListener.java

```diff
@@ -94,7 +93,7 @@ public MotanServiceEventListener(final PropertiesConfig clientConfig, final Shen
     @Override
     public void onApplicationEvent(final ContextRefreshedEvent contextRefreshedEvent) throws BeansException {
         applicationContext = contextRefreshedEvent.getApplicationContext();
-        Map<String, Object> beans = applicationContext.getBeansWithAnnotation(ShenyuMotanClient.class);
+        Map<String, Object> beans = applicationContext.getBeansWithAnnotation(ShenyuMotanService.class);
```

Comment:
why change this ?  `applicationContext.getBeansWithAnnotation(ShenyuMotanClient.class);` can get more beans, if changed, some bean with `@ShenyuMotanClient` will not got.

Reply:
I think you don't need to modify `MotanServiceEventListener`